test/unit/test_streaming_client_stream_decryptor.py

```diff
@@ -364,142 +365,6 @@ def test_GIVEN_verification_key_AND_has_mpl_AND_has_MPLCMM_WHEN_read_header_THEN
             algorithm=self.mock_header.algorithm, encoded_point=mock_b64encoding()
         )
 
-    @patch("aws_encryption_sdk.streaming_client.derive_data_encryption_key")
-    @patch("aws_encryption_sdk.streaming_client.DecryptionMaterialsRequest")
-    @patch("aws_encryption_sdk.streaming_client.Verifier")
-    # Given: no MPL
-    @pytest.mark.skipif(HAS_MPL, reason="Test should only be executed without MPL in installation")
-    def test_GIVEN_decrypt_config_has_ec_WHEN_read_header_THEN_calls_decrypt_materials_with_reproduced_ec(
-        self,
```

Comment:
Just wondering why we are deleting these?

Reply:
This might be a diff issue... let me double check

Reply:
Ah! This was a merge conflict issue. I committed this test when I shouldn't have.
The behavior from this test is no longer valid.
This test would assert that encryption context on decrypt works for native (ESDK-Python) CMMs.
We decided that encryption context on decrypt would not be supported in this [thread](https://github.com/aws/aws-encryption-sdk-python/pull/645#discussion_r1569435020).
The new behavior validating encryption context on decrypt raises an exception is present [here](https://github.com/aws/aws-encryption-sdk-python/blob/mpl-reviewed/test/unit/test_streaming_client_stream_decryptor.py#L306).